plugins/woocommerce/includes/class-wc-checkout.php

```diff
@@ -783,6 +783,9 @@ public function get_posted_data() {
 							$value = wc_sanitize_textarea( $value );
 							break;
 						case 'password':
+							if ( $data[ 'createaccount' ] && 'account_password' === $key ) {
+								$value = wp_slash( $value ); // Passwords are encrypted with slashes on account creation, so we need to slash here too.
```

Comment:
> Ok, but can we at least track where the unslash is happening @wavvves? I cannot see it happening when wc_create_new_customer is called. It would be useful to confirm where the unslash first happens so it can be documented clearly in the code why the slash is needed. Or we can remove the unslash at source.

@mikejolley unslashing is being done here on L767. Instead of adding an exception there I think it would be more readable and self-contained to be re-slashed on the switch. The tiny performance hit of just avoiding unslashing can be neglected IMO.
What do you think?

Reply:
@mikejolley Here is the direct link to it: https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/includes/class-wc-checkout.php#L767